lib/active_admin/view_helpers/method_or_proc_helper.rb

```diff
@@ -60,6 +65,7 @@ def call_method_or_proc_on(receiver, *args)
       if options[:exec]
         instance_exec(receiver, *args, &symbol_or_proc)
       else
+        # exec: false is deprecated.  TODO: raise if this branch called.
```

Comment:
Should we throw a deprecation warning here? I'll make a reminder for us to remove anything deprecated once we start v2 in master.